src/consent-manager/preference-dialog.tsx

```diff
@@ -85,6 +86,15 @@ const SaveButton = styled(Button)`
   margin-left: 8px;
 `
 
+const categoryPlaceholder = '[CATEGORY_NAME]'
+const replaceWith = (category: string, phrase: string) =>
+  phrase.replace(categoryPlaceholder, category)
+const categoriesList = {
```

Comment:
Could you please describe benefits of using this dictionary?

Reply:
Only for taking category's names under one roof. So if we need to expand/modify categories in the future, we can look at them in one place and add/change them here without surfing through the file.

As I see you're offering to simplify it and use straightforward from translations like:
```
 replaceWith(
    translations.functional_category,
    translations.allow_category_tracking
)
```

correct?

Reply:
For now not unified approach is used, for me simplified one (^^) looks better as an option for unifying

Reply:
done